app/services/startup_feedback_module/email_service.rb

```diff
@@ -2,14 +2,15 @@ module StartupFeedbackModule
   class EmailService
     include Loggable
 
-    def initialize(startup_feedback)
+    def initialize(startup_feedback, grading_details=nil)
```

Comment:
With reference to the comment in the `create_grading` mutation, the second argument can just be a boolean flag which decides whether grading should be included in the email.
```ruby
def feedback_as_email(startup_feedback, include_grades: false)
```

Based on this flag, the grading details computation can be performed in the `StartupMailer`.